src/aliceVision/hdr/brackets.cpp

```diff
@@ -273,10 +273,11 @@ int selectTargetViews(std::vector<std::shared_ptr<sfmData::View>>& out_targetVie
 
     for (auto& group : groups)
     {
-        // Set the ldr ancestors id
+        // Set the ldr ancestors
         for (auto v : group)
         {
             group[targetIndex]->addAncestor(v->getViewId());
+            group[targetIndex]->addAncestorImage(v->getViewId());
```

Comment:
I think, we need only one. addAncestor should be enough.

Reply:
Correction done.